test/integration/crud/bulk_execute_operation.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+
+describe('Bulk executeOperation', () => {
+  let client;
+  beforeEach(async function () {
+    client = await this.configuration.newClient({ monitorCommands: true }).connect();
+  });
+  afterEach(async () => {
+    await client?.close();
+  });
+
+  it('should use the same session for every operation', async () => {
```

Comment:
should we do 2 cases here, for completeness, both for ordered and unordered ops? since the change was to common?

Reply:
The difference between ordered and unordered is the ability to affect grouping of the operations, I don't think it adds any additional coverage, the shim class isn't a "real" operation that get's executed it's for us to pass the options an properties along in the right shape for executeCommands to build each actual operation

Reply:
I think it's low risk enough to be fine in this case, but I think generally as a best practice, whenever we are using an integration test and not a unit test to make sure the class works correctly, and whenever there are two or more distinct ways to access the code under test in an integration setting, we should try to test all the pathways (in this case, it would be to ensure that the overrides from the specific bulk class don't affect the session attachment from the base class)